extension/locale/sv-SE/vimfx.properties

```diff
@@ -0,0 +1,85 @@
+help_command_marker_focus=Fokusera element
+help_command_yank=Kopiera länk till den nuvarande sidan till urklipp
+help_command_reload=Ladda om den nuvarande sidan
+help_command_reload_force=Ladda om den nuvarande sidan och alla resurser (js, css, img)
+help_command_reload_all=Ladda om sidor i alla flikar
+help_command_reload_all_force=Ladda om sidor i alla flikar, inklusive alla resurser (js, css, img)
+help_command_stop=Stoppa inladdning av den nuvarande sidan
+help_command_stop_all=Stoppa inladdning av sidor i alla flikar
+
+help_section_nav=Navigera sidan
+help_command_scroll_to_top=Scrolla högst upp på sidan
+help_command_scroll_to_bottom=Scrolla längst ned på sidan
+help_command_scroll_down=Scolla ned
+help_command_scroll_up=Scrolla upp
+help_command_scroll_left=Scrolla åt vänster
+help_command_scroll_right=Scrolla åt höger
+help_command_scroll_half_page_down=Scrolla en halv sida ned
+help_command_scroll_half_page_up=Scrolla en halv sida upp
+help_command_scroll_page_down=Scrolla en hel sida ned
+help_command_scroll_page_up=Scrolla en hel sida upp
+
+help_section_tabs=Arbeta med flikar
+help_command_open_tab=Öppna ny tom flik
+help_command_tab_prev=Gå till föregående flik
+help_command_tab_next=Gå till nästa flik
+help_command_tab_move_left=Flytta nuvarande flik åt vänster
+help_command_tab_move_right=Flytta nuvarande flik åt höger
+help_command_home=Gå till Hemsidan
+help_command_tab_first=Gå till den första fliken
+help_command_tab_first_non_pinned=Gå till den första icke-fästa fliken
+help_command_tab_last=Gå till den sista fliken
+help_command_close_tab=Stäng den nuvarande fliken
+help_command_restore_tab=Återställ senast stängd flik
+
+
+help_section_browse=Surfande
+help_command_follow=Öppna en länk på den nuvarnade sidan
+help_command_follow_in_tab=Öppna en länk på den nuvarande sidan i en ny flik
+help_command_follow_multiple=Öppna flera länkar på den nuvarande sidan
+help_command_follow_previous=Gå till föregående sida
+help_command_follow_next=Gå till nästa sida
+help_command_go_up_path=Gå upp en nivå i URL hierarkin
+help_command_go_to_root=Gå till roten av URL hierarkin
+help_command_back=Gå tillbaka i historiken
+help_command_forward=Gå framåt i historiken
+
+help_section_misc=Övrigt
+help_command_help=Visa denna rutan
+help_command_Esc=Gå till normalläge (tar bort etikettsmarkeringar, går ur inmatningsläge) eller avfokusera/stäng aktivt element
+help_command_find=Gå till sökläge
+help_command_find_hl=Gå till sökläge och markera alla sökresultat
+help_command_find_next=Gå till nästa hittade sökresultat
+help_command_find_prev=Gå till föregående hittade sökresultat
+help_command_insert_mode=Gå till inmatningsläge: Ignorera alla kommandon
+help_command_dev=Öppna utvecklarverktygsfältet
+
+help_title=Tangentbordskommandon
+help_version=Version
+
+help_overlapping_hints=Kan Du inte se en länketikett då den täcks över av en annan? Testa att klicka Mellanslag eller Shift-Mellanslag!
```

Comment:
Sure, I don't really have anything against it and maybe someone else think like you do. I'll go with that.